pkg/volume/rbd/rbd_util.go

```diff
@@ -101,7 +101,7 @@ func (util *RBDUtil) AttachDisk(rbd rbd) error {
 	// mount it
 	globalPDPath := rbd.manager.MakeGlobalPDName(rbd)
 	mountpoint, err := rbd.mounter.IsMountPoint(globalPDPath)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
I would add a note saying that the mountpoint not existing is a contingency.

Do you want this method to continue running if there's already a mountpoint for this volume?